consensus/enclave/impl/src/lib.rs

```diff
@@ -1017,8 +1037,10 @@ mod tests {
                 .unwrap();
 
             // Check that the context we got back is correct.
+            let (expected_priority, _) = ct_u64_divide(tx.prefix.fee, Mob::MINIMUM_FEE / 128);
```

Comment:
Let's use a constant here
```suggestion
            let (expected_priority, _) = ct_u64_divide(tx.prefix.fee, Mob::MINIMUM_FEE / SMALLEST_MINIMUM_FEE);
```